src/component/mxgraph/shape/render/IconPainter.ts

```diff
@@ -233,6 +233,33 @@ export default class IconPainter {
     canvas.fillAndStroke();
   }
 
+  // implementation adapted from https://thenounproject.com/term/triangle/2452089/
+  public paintTriangleIcon({ c, ratioFromParent, shape: { x, y, w, h }, icon }: PaintParameter): void {
+    const canvas = new BpmnCanvas({
+      mxCanvas: c,
+      shapeConfiguration: { x, y, w, h },
+      iconConfiguration: {
+        originalSize: { height: 735, width: 849 },
+        ratioFromShape: ratioFromParent,
+        style: icon,
+      },
+    });
+    c.setStrokeWidth(StyleDefault.STROKE_WIDTH_THIN); // TODO should be done via icon style, but in that case, the strokeWidth change the ratio from shape
```

Comment:
It's normal that the strokeWidth change the ratio from the shape, because MxGraph increase the size of the icon when the stroke width is greater than 1.
So to keep consistency and have the same size of an icon (whatever its stroke width), the ratio from the shape change 😉